src/main/java/it/gov/pagopa/pu/sil/service/debtposition/DebtPositionCheckoutService.java

```diff
@@ -81,6 +95,24 @@ public String redirectToCheckout(UserInfo loggedUser, String accessToken) {
     return checkoutUrl;
   }
 
+  public String composeDebtPositionsCheckoutUrl(Long organizationId, String iuvs, String callbackUrl, String orgFiscalCode, String accessToken) {
+    Map<String, Object> sessionData = new HashMap<>();
+    sessionData.put(CALLBACK_URL_SESSION_DATA_KEY, callbackUrl);
+
+    LimitedTokenRequest limitedTokenRequest = LimitedTokenRequest.builder()
+      .app(PU_SIL_SCOPE)
+      .organizationId(organizationId)
+      .resource(CHECKOUT_RESOURCE)
+      .resourceId(iuvs)
+      .sessionData(sessionData)
+      .expireInSeconds(24L * 60 * 60)
+      .singleUsage(false)
+      .build();
+    AccessToken limitedToken = authorizationService.requestLimitedToken(limitedTokenRequest, accessToken);
+
+    return "/organization/%s/checkout?limitedToken=%s".formatted(orgFiscalCode, limitedToken.getAccessToken()); // TODO: fix url
```

Comment:
here you should configure the public URL of the service which is exposing this API:
AS result, the API should return a complete URL: as test, if you follow the link, you should be redirected towards checkout with the cart filled with the selected IUVs